src/main/resources/text/helpForAllCommands.txt

```diff
@@ -8,6 +8,12 @@ contact find                Find contacts given search keyword(s)
 contact delete              Deletes a contact                                           contact delete INDEX
 contact sort                Sorts contact list                                          contact sort KEYWORD [ORDER]
 
+Archive commands
+----------------------
+archive add                 Sends a contact to the archive                              contact archive INDEX
```

Comment:
the example usage is not updated?

Reply:
Whoops my bad. Will update on another feature branch.